tests/components/vera/test_init.py

```diff
@@ -33,16 +33,16 @@ async def test_init(
         hass=hass,
         controller_config=new_simple_controller_config(
             config={CONF_CONTROLLER: "http://127.0.0.1:111"},
-            config_from_file=False,
+            config_source=ConfigSource.CONFIG_FLOW,
             serial_number="first_serial",
             devices=(vera_device1,),
         ),
     )
 
     entity_registry = await hass.helpers.entity_registry.async_get_registry()
     entry1 = entity_registry.async_get(entity1_id)
-
     assert entry1
+    assert entry1.unique_id == "vera_first_serial_1"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)